lib/AppInfo/Application.php

```diff
@@ -69,7 +69,7 @@ public function __construct(array $urlParams = []) {
 	}
 
 	public function register(IRegistrationContext $context): void {
-		$context->registerEventListener(BeforeNodeDeletedEvent::class, FileListener::class);
+		$context->registerEventListener(NodeDeletedEvent::class, FileListener::class);
```

Comment:
Curious, why this change? We might need it in recognize as well

Reply:
just thought we should not delete the file from the backend until the deletion is done in the server

Reply:
I think the reason for using Before\* was because the handler below needs the node to still be available in the file system, I worry that this change will break that.

Reply:
oh you're right. I only tested with files for which the node was in the event. It will fail for folders.
nice catch!